kubernetes/template-generator/template_generator.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/env python
+
+import re
+import yaml
+import json
+
+from container_os import *
```

Comment:
the exception might be if the class was implemented poor and the author would force you to do this :-D  i hope it's the case  here :)